### PR TITLE
Add Okta Provider

### PR DESCRIPTION
**Pull request by chaseconey**

### CODE REVIEW

**Comment by chaseconey** [comment]
This one is a little bit different because it requires a URL custom to your company. Not sure if just creating another config/env setting for that is ok or not. Please advise.

File: src/Okta/Provider.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    protected function getOktaUrl()
+    {
+        return config('services.okta.url');
```
Comment (faustbrian) [inline src/Okta/Provider.php]: Use https://github.com/SocialiteProviders/Manager/blob/master/src/ConfigTrait.php#L46-L52 for additional configuration keys.

**Comment by chaseconey** [comment]
@faustbrian I think that should do it. Let me know what you think.

**Comment by faustbrian** [comment]
Send a PR to https://github.com/SocialiteProviders/socialiteproviders.github.io if you want to add it to the docs.

**Pull request merged**